packages/commonwealth/client/scripts/views/components/FeedDiscovery/FeedDiscovery.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import {Helmet} from "react-helmet";
```

Comment:
it look's like your prettier is not configured properly? @kurtisassad any advices here? Do we still need to run any yarn command or? It is crucial to set it up correctly at the beginning

Reply:
Yeah strange. It should be auto-installed via husky on yarn install. Do you notice any messages talking about running prettier when committing @burtonator ?

Reply:
@kurtisassad no messages regarding prettier but my IDE isn't showing eslint rules either.  

If these are critical though we should be running prettier during the commit as plan B to prevent bad merges.

I'll take a look now though.

Reply:
Ah I see the issue here. I think you need to run `yarn install` from root, not from any sub-directory @burtonator